privaterelay/templates/includes/header.html

```diff
@@ -97,5 +97,5 @@
       {% endif %}
     </div>
   </nav>
-  <fluent class="bento-strings" data-bento-button-title="{% ftlmsg 'bento-button-title' %}" data-fx-makes-tech="{% ftlmsg 'fx-makes-tech' %}" data-made-by-mozilla="{% ftlmsg 'made-by-mozilla' %}" data-fx-desktop="{% ftlmsg 'fx-desktop' %}" data-fx-lockwise="{% ftlmsg 'fx-lockwise' %}" data-fx-mobile="{% ftlmsg 'fx-mobile' %}" data-fx-monitor="{% ftlmsg 'fx-monitor' %}" data-fx-pocket="{% ftlmsg 'fx-pocket' %}" data-bento-button-close-label="{% ftlmsg 'bento-button-close-label' %}"></fluent>
+  <fluent class="bento-strings" data-bento-button-title="{% ftlmsg 'bento-button-title' %}" data-fx-makes-tech="{% ftlmsg 'fx-makes-tech' %}" data-made-by-mozilla="{% ftlmsg 'made-by-mozilla' %}" data-fx-desktop="{% ftlmsg 'fx-desktop' %}" data-fx-lockwise="{% ftlmsg 'fx-lockwise' %}" data-fx-mobile="{% ftlmsg 'fx-mobile' %}" data-fx-monitor="{% ftlmsg 'fx-monitor' %}" data-fx-pocket="{% ftlmsg 'fx-pocket' %}" data-fx-vpn="Mozilla VPN" data-bento-button-close-label="{% ftlmsg 'bento-button-close-label' %}"></fluent>
```

Comment:
I don't think product names are usually translated, but since the other product names were added to the Fluent files, maybe this one would need to be too?

Reply:
Ah, I see that's because https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/51 isn't merged yet :)